### PR TITLE
fix(troubleshooting sections): make them consistent

### PR DESCRIPTION
**Pull request by NathalieC**
* Performance troubleshooting page

### CODE REVIEW

File: modules/process/pages/generate-pdf-from-an-office-document.adoc [hunk 1]
```diff
@@ -38,6 +38,16 @@ Rendering issues (at worst the element will not be visible in the generated PDF;
 * HTML conversion generates a single html file. For that matter, linked resources (like images, ...) are not supported. If you have images, use PDF conversion instead. +
 This connector is build upon *XdocReport* third party library, for more information check their Wiki at https://github.com/opensagres/xdocreport/wiki
 
-== Troubleshooting
-
-You may encounter memory issues using this connector as it has substantial size of dependencies. To workaround those issues you may increase the maximum memory allocation value of the Tomcat server JVM. Go to Preferences \-> Server settings \-> Increase the Tomcat Maximum memory allocation (`-Xmx` JVM argument) value to *1024* (512 by default). Do not forget to use the same value when configuring your production Bundle.
+== image:ROOT/assets/images/images/troubleshooting.png[troubleshooting-icon] Troubleshooting
```
Comment (tbouffard) [inline modules/process/pages/generate-pdf-from-an-office-document.adoc]: The current syntax doesn't work, try this one but without guarantee 😺 
```suggestion
== image:ROOT:images/troubleshooting.png[troubleshooting-icon] Troubleshooting
```

File: modules/process/pages/generate-pdf-from-an-office-document.adoc [hunk 2]
```diff
@@ -38,6 +38,16 @@ Rendering issues (at worst the element will not be visible in the generated PDF;
...
+
+*Symptom*: +
+Users may encounter delays (long response times) when using this connector.
+*Cause:* +
+Memory issue as it has substantial size of dependencies. 
+*Potential solution:* +
+You may increase the maximum memory allocation value of the Tomcat server JVM. +
+To do so:
+ * Go to Preferences \-> Server settings \
```
Comment (tbouffard) [inline modules/process/pages/generate-pdf-from-an-office-document.adoc]: Here you can you the `menu` macro: https://docs.antora.org/antora/2.3/asciidoc/ui-macros/#set-the-experimental-attribute
You can check https://github.com/bonitasoft/bonita-doc/pull/1826/files for a quick how-to (including how to declare the `experimental` AsciiDoc attribute within the page)

**Comment by alachambre** [comment]
@NathalieC you are doing it again. 17 commits, 1 month alive, no rebase. 
If you don't have time for such contributions don't do those contributions. 
Keep in mind that this time I won't be able to help you to merge it at the end.

File: modules/process/pages/generate-pdf-from-an-office-document.adoc (hunk 2 above)
Comment (NathalieC) [reply modules/process/pages/generate-pdf-from-an-office-document.adoc]: Thanks. I'm not sure about the added value compared to just write the bread crumbs by hand.
But I'm curious, and ready to try.

**Comment by alachambre** [comment]
@NathalieC tu as remarqué que ta pr est rouge depuis le début? Ca ne t'inquiète pas plus que ça? ^^

**Review by alachambre — APPROVED** [review]

**Pull request merged**